### DISCUSSION TITLE
bpmn/bpmn-js-vue-custom-viewer-demo-code-source

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#62] [Announcements]
# bpmn/bpmn-js-vue-custom-viewer-demo-code-source

最好用的流程编辑器bpmn-js系列文章

https://blog.ops-coffee.cn/bpmn/bpmn-js-vue-custom-viewer-demo-code-source.html

**Comment by melody-submit** [comment]
very good
